src/api/utils.ts

```diff
@@ -18,6 +20,8 @@ const defaultConfig = {
   ssl: false,
   sslcert: null,
   sslkey: null,
+  rsapublickey: key.exportKey('public'),
```

Comment:
please rename to `rsaPublicKey` and `rsaPrivateKey` (or `privateKey`, not sure how it is correct).